pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -26,44 +26,61 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller_wmc")
+const (
+	// ControllerName is the name of the WMC controller
+	ControllerName = "windowsmachineconfig-controller"
+)
 
-/**
-* USER ACTION REQUIRED: This is a scaffold file intended for the user to modify with their own Controller
-* business logic.  Delete these comments after modifying this file.*
- */
+var log = logf.Log.WithName("controller_wmc")
 
 // Add creates a new WindowsMachineConfig Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager) error {
-	return add(mgr, newReconciler(mgr))
+	reconciler, err := newReconciler(mgr)
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("could not create %s reconciler", ControllerName))
```

Comment:
We can use `errors.Wrapf()` (https://godoc.org/github.com/pkg/errors#Wrapf) to represent this. Need to make changes in multiple places.